resource-timing/single-entry-per-resource.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE HTML>
+<meta charset=utf-8>
+<meta name="timeout" content="long">
+<title>One resource when reusing data</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="../performance-timeline/performanceobservers.js"></script>
+<script>
+  async_test(function (t) {
+    function path(pathname) {
+      var filename = pathname.substring(pathname.lastIndexOf('/')+1);
+      return pathname.substring(0, pathname.length - filename.length);
+    }
+    var img_location = document.location.origin + path(document.location.pathname)
+       + "resources/blue.png";
+    var observer = new PerformanceObserver(
+        t.step_func(function (entryList, obs) {
```

Comment:
Perhaps unlikely, but this test wouldn't catch an implementation that fires two PO callbacks, one for each img, right? Would it make sense to accumulate the entries and verify at onload that there is only one?